.github/workflows/ci.yml

```diff
@@ -155,9 +156,10 @@ jobs:
           HELICONE_CACHE_ENABLED: true
           HELICONE_PROPERTY_AGENT: ${{ matrix.agent-name }}
 
-      - name: Upload logs as artifact
+
+      - name: Upload reports
         if: always()
         uses: actions/upload-artifact@v3
         with:
-          name: gpt-engineer-projects
-          path: agent/gpt-engineer/projects
```

Comment:
I removed this because it depends on the agent we benchmark. I will add it back for each agent so we can see their workspace, in order to debug